tasks/src/unconditional-image-generation/data.ts

```diff
@@ -0,0 +1,55 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "NFT images dataset of 'cryptopunks' for unconditional image generation.",
+			id:          "huggingnft/cryptopunks",
+		},
+		{
+			description: "NFT images dataset of 'alpacadabraz' for unconditional image generation.",
+			id:          "huggingnft/alpacadabraz",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Input",
+				content: "",
+				type:    "text",
+			},
+		],
+		outputs: [
+			{
+				filename: "unconditional-image-generation-output.jpeg",
+				type:     "img",
+			},
+		],
+	},
+	metrics: [
+		{
+			description: "The inception score (IS) evaluates the quality of image generative models by using the output of a pre-trained classifier (InceptionV3). It measures that the generated images are diverse (the model predictions are evenly distributed across all possible labels) and distinct (the model confidently predicts a single label for each image).",
```

Comment:
```suggestion
			description: "The inception score (IS) evaluates the quality of image generative models). It measures the diversity of the generated images (the model predictions are evenly distributed across all possible labels) and distinct (the model confidently predicts a single label for each image).",
```
nit 🙂

Reply:
Great! I am just noticing an extra ")" in your suggestion, so I'll change it in my PR with the other corrections if that's ok for you.